core/domain/feedback_services.py

```diff
@@ -291,12 +308,39 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_email_task(user_id)
 
 
-def send_feedback_message_email(exploration_id, thread_id, message_id):
+def update_feedback_email_retries(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+    time = (datetime.datetime.utcnow() - model.created_on).seconds
```

Comment:
Actually created_on is property of base model, so it applies to all models created by inheriting base model. And it has auto_add_now flag set to true. So it actually depends on gae or server machine's time may be. I do not have idea. I will have to look into it. 

When I used now instead of utcnow and ran backend tests, a info appeared in log that I should use utcnow. So I though that it is already configured in system.